README.md

```diff
@@ -167,6 +167,30 @@ The ID3 supported frames supported are (see the enum `FrameName`):
 * `.TrackPosition`, track position frame
 * `.Genre`, the genre frame
 * `.AttachedPicture(_ pictureType: ID3PictureType)` the attached picture frame
+* `Comment`
```

Comment:
`Comment` should be removed.

Reply:
Will do. Did you mean to close the PR? Should I start a new one once I've implemented the rest of the changes?